### ISSUE TITLE
iTerm style fullscreen

### ISSUE THREAD

**Issue by justmytwospence** [railwaycat/emacs-mac-port#93]
The readme makes it sound like it is possible to get an "iTerm2" style fullscreen: no titlebar on the emacs OSX window, but visible OSX menubar at the top. I can't seem to produce this configuration, however. Is it possible?

**Comment by 4lex1v** [comment]
@justmytwospence Not sure, but are you talking about [this](https://github.com/nikhilunni/BorderlessEmacs) trick?

**Comment by justmytwospence** [comment]
Yeah basically! iTerm only has a setting for when its in fullscreen: like this:

<img width="1440" alt="screen shot 2015-08-20 at 1 11 46 pm" src="https://cloud.githubusercontent.com/assets/2817686/9394404/2ff73d80-473d-11e5-8ba4-ae1d34978e1d.png">

It would be an awesome feature for this port - the one thing I'm really pining for.

**Comment by khalidchawtany** [comment]
Nice trick :+1:
